pyanaconda/exception.py

```diff
@@ -248,7 +248,8 @@ def initExceptionHandling(anaconda):
                  "/tmp/program.log", "/tmp/storage.log", "/tmp/ifcfg.log",
                  "/tmp/dnf.librepo.log", "/tmp/hawkey.log",
                  "/tmp/lvm.log", iutil.getSysroot() + "/root/install.log",
-                 "/proc/cmdline", "/root/lorax-packages.log"]
+                 "/proc/cmdline", "/root/lorax-packages.log",
+                 "/tmp/blivet-gui-utils.log"]
```

Comment:
Oh, so it's name of the process ? I guess that makes sense - I think it would still be better if the process & log file were called Blivet GUI daemon/Blivet GUI backend (or something similar), but I thats probably a substantial amount of work for little gain, at least for now.